### ISSUE TITLE
Build error of version 3.4.1 using blueprint-compiler 0.8.1

### ISSUE THREAD

**Issue by BenediktBroich** [liferooter/textpieces#130]
Building textpieces v3.4.1 with blueprint-compiler v0.8.1 results in the following Error:
(I am updating [blueprint-compiler for nixos](https://github.com/NixOS/nixpkgs/pull/238303). [Arch linux seams to have the same issue](https://aur.archlinux.org/packages/textpieces#comment-915266).

```
upgrade: Use type syntax here (introduced in blueprint 0.8.0)
at ../resources/ui/CustomToolPage.blp line 8 column 10:
   8 |template TextPiecesCustomToolPage : Box {
     |         ^

upgrade: Use the '$' extern syntax introduced in blueprint 0.8.0
at ../resources/ui/CustomToolPage.blp line 17 column 18:
  17 |      clicked => go_back();
     |                 ^

upgrade: Use the '$' extern syntax introduced in blueprint 0.8.0
at ../resources/ui/CustomToolPage.blp line 36 column 18:
  36 |      clicked => delete_tool();
     |                 ^

upgrade: Use the '$' extern syntax introduced in blueprint 0.8.0
at ../resources/ui/CustomToolPage.blp line 37 column 18:
  37 |      clicked => go_back();
     |                 ^

upgrade: Use the '$' extern syntax introduced in blueprint 0.8.0
at ../resources/ui/CustomToolPage.blp line 43 column 3:
  43 |  .TextPiecesToolSettings tool_settings {
     |  ^

error: Cannot convert 2.5 to integer
at ../resources/ui/Editor.blp line 45 column 26:
  45 |          margin-bottom: 2.5;
     |                         ^

ninja: build stopped: subcommand failed.
```

**Comment by akhial** [comment]
Fixed this locally by changing margin from `2.5` to `3`.
I think the new version changed margins to only allow integers?

Two files are affected `Editor.blp` and `SearchBar.blp`.

**Comment by liferooter** [comment]
Fixed in Text Pieces 4.0, which will be released soon
